src/screens/AirdropRequirements/AirdropRequirementsScreen.tsx

```diff
@@ -23,8 +24,22 @@ type Props = {
 } & ActionProps;
 
 class AirdropRequirementsScreen extends Component<Props> {
+  removeAirdropScreensFromHistory = () => {
+    this.props.navigation.dispatch(state => {
+      const routes = state.routes.filter(r => r.name !== Route.AirdropThankYou && r.name !== Route.AirdropRequirements);
```

Comment:
It's up to you, but maybe it'd be nicer to write it this way
```
const routes = state.routes.filter(r => ![Route.AirdropThankYou, Route.AirdropRequirements].includes(r.name));
```
?

Reply:
code removed.